Sources/ArgumentParser/Parsing/Name.swift

```diff
@@ -53,6 +53,10 @@ extension Name {
       return n
     }
   }
+  
+  var isShort: Bool {
+    return self == .short(self.valueString.first!)
```

Comment:
One last nit — this should use pattern-matching on `self` instead of an equality comparison. We don't need to be accessing `valueString` for this.

Reply:
Yes, you are right. Just changed that.